ouster-ros/launch/driver.launch.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2023 Ouster, Inc.
+#
+
+"""Launch a sensor node along with os_cloud and os_"""
+
+from pathlib import Path
+import launch
+import lifecycle_msgs.msg
+from ament_index_python.packages import get_package_share_directory
+from launch_ros.actions import Node, LifecycleNode
+from launch.actions import (DeclareLaunchArgument, IncludeLaunchDescription,
+                            RegisterEventHandler, EmitEvent, LogInfo)
+from launch.conditions import IfCondition
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
+from launch.events import matches_action
+from launch_ros.events.lifecycle import ChangeState
+from launch_ros.event_handlers import OnStateTransition
+
+
+def generate_launch_description():
+    """
+    Generate launch description for running ouster_ros components separately each
+    component will run in a separate process).
+    """
+    ouster_ros_pkg_dir = get_package_share_directory('ouster_ros')
+    default_params_file = \
+        Path(ouster_ros_pkg_dir) / 'config' / 'parameters.yaml'
```

Comment:
dirver_params.yaml ?

Reply:
Yes, this should be `driver_params.yaml`, I will change it.